### PR TITLE
Added the latest options for `select` fields.

### PR DESCRIPTION
**Pull request by bobdenotter**

### CODE REVIEW

File: source/howto/select-field-type.md [hunk 1]
```diff
@@ -160,5 +160,26 @@ comma. The following example will display the 'first name' and 'last name' of ea
 ```
 
 
+If the list is growing longer, there are a few ways to make it more manageable:
 
+```apache
```
Comment (GawainLynch) [inline source/howto/select-field-type.md]: Shouldn't these be `yaml`

**Comment by GawainLynch** [comment]
No real objections to this, but just flagging it. Would it be better to do images and bits in English?

File: source/howto/select-field-type.md (hunk 1 above)
Comment (bobdenotter) [inline source/howto/select-field-type.md]: highlight.js doesn't support `yml`, and `apache` looks nicer than whatever it picks as a fallback. ;-)

Comment (GawainLynch) [inline source/howto/select-field-type.md]: Ah, highlight! Oh, look, :koala:

**Comment by bobdenotter** [comment]
Yeah, i was lazy. Had that example lying around. Think it's an issue? If so, i'll make an example screenshot in english.

**Comment by GawainLynch** [comment]
Nah, just flagging it. Someone's CDO will kick in a some point anyway.

**Pull request merged**